apps/emqx_bridge/src/emqx_bridge.erl

```diff
@@ -407,6 +410,33 @@ if_only_to_toggole_enable(OldConf, Conf) ->
         {_, _, _} -> false
     end.
 
+-spec get_basic_usage_info() ->
+          #{ ids => [BridgeID]
+           , count_by_type =>
+                 #{ BridgeType => non_neg_integer()
+                  }
+           } when BridgeID   :: binary(),
+                  BridgeType :: binary().
+get_basic_usage_info() ->
+    lists:foldl(
+      fun(#{resource_data := #{config := #{enable := false}}}, Acc) ->
+              Acc;
+         (#{name := BridgeName, type := BridgeType}, Acc) ->
+              IDs = maps:get(ids, Acc),
+              BridgeID = bridge_id(BridgeType, BridgeName),
```

Comment:
since the name is assigned by user, it can be arbitrary string
reporting it in telemetry will make this dimension explode ?

Reply:
i.e. maybe report just the type.